main/services/media.js

```diff
@@ -43,7 +43,7 @@ async function downloadAndSave(media, url) {
   const { protocol } = parse(url)
   const isRemote = protocol && protocol.startsWith('http')
   const source = isRemote
-    ? got.stream(url, { timeout: 3000 })
+    ? got.stream(url, { timeout: { send: 3e3, response: 3e3, lookup: 500 } })
```

Comment:
Codacy found an issue: [The numeric literal '3e3' will have at different value at runtime.](https://app.codacy.com/gh/feugy/melodie/pullRequest?prid=6381868)